tests/managers_regress/tests.py

```diff
@@ -167,3 +167,105 @@ def test_field_can_be_called_exact(self):
         related = RelatedModel.objects.create(exact=False)
         relation = related.test_fk.create()
         self.assertEqual(related.test_fk.get(), relation)
+
+    @isolate_apps('managers_regress')
+    def test_parent_custom_manager_is_not_available_for_child_with_multi_table_inheritance(self):
+        class BaseModel(models.Model):
+            pass
+
+        class TestModel(BaseModel):
+            pass
+
+        self.assertEqual(hasattr(TestModel, 'test_objects'), False)
```

Comment:
This test name mentions multi-table inheritance but the body of the test has nothing to do with it.